src/templates/doe_workflow_central_ducted_split_heat_pump.mdx

```diff
@@ -354,11 +354,11 @@
 
             
             {
-                props.metadata.attachments['manual_j_file'] || props.data.comment_manual_j ?
+                props.metadata.attachments[props.jobId]?.manual_j_file || props.data.comment_manual_j ?
                 <>
                 <h2> Manual J </h2>
                 {
-                props.metadata.attachments['manual_j_file'] ? <p>Please see the PDF containing the Manual J report attached.</p> : null
+                 props.metadata.attachments[props.jobId]?.manual_j_file  ? <p>Please see the PDF containing the Manual J report attached.</p> : null
```

Comment:
Let's reorder this sentence to sound a bit better:
"Please see the attached PDF containing the Manual J report."

Reply:
Updated the text accordinlgy.